test/FIFOMarket.test.ts

```diff
@@ -960,6 +906,62 @@ describe('FIFOMarket', () => {
         expect(noriFinalNoriBalance).to.equal(noriInitialNoriBalance);
         expect(await certificate.balanceOf(buyer.address, 0)).to.equal(0);
       });
+      // TODO disallow listing removals with 0 balance in the first place.
```

Comment:
I did yeah... we have a ticket in place to address the fact that removal amounts of 0 should be prevented upstream before a certificate is being minted. such that this test shouldn't even be viable because the place where it lists removals it lists them with an amount of 0, something that we will prevent in the future.  the test broken because I removed the revert statement when minting certificates and instead of fixing it I think we should just fix it when we make the final fix